### PR TITLE
Ana/livepeer explore mode frontend

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Just a couple of changes to improve frontend to allow more networks for the signin (not only validate for bech32 addresses, now also for hex ones) and also regarding undelegations (it wasn't properly filtering before for date, searching for the really pending ones).

Few important changes to host more types of networks.

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Another solution would be to make a specific query in the API for `pendingUndelegations` only... Maybe better than this solution :thinking:

File: src/components/common/TmSessionExplore.vue [hunk 1]
```diff
@@ -142,13 +142,31 @@ export default {
     exploreWith(address) {
       this.address = address
       this.onSubmit()
+    },
+    isEthereumAddress(address) {
```
Comment (faboweb) [inline src/components/common/TmSessionExplore.vue]: very nicely handled

File: src/components/staking/Undelegations.vue [hunk 2]
```diff
@@ -26,6 +32,13 @@ export default {
   computed: {
     ...mapGetters([`address`, `network`])
   },
+  methods: {
+    filterOnlyPendingUndelegations(undelegations) {
```
Comment (faboweb) [inline src/components/staking/Undelegations.vue]: sweet! I agree though, that we should put this logic into the API to make the FE stupider

**Comment by faboweb** [comment]
Missing some coverage for handling ethereum addresses. Tip: click on `codecov/patch` > `Details` > any red box. Now you see the not covered lines.

**Comment by Bitcoinera** [comment]
> Missing some coverage for handling ethereum addresses. Tip: click on `codecov/patch` > `Details` > any red box. Now you see the not covered lines.

Yes. I also want to add to this PR the separate call for `selfStake`, that is why I didn't pay so much attention to this yet. Should have changed the title to WIP to make it clear.

**Comment by Bitcoinera** [comment]
I'd appreciate if we could merge this asap. Otherwise it is a bit clumsy to sign in while I'm testing. 
Thank you

**Comment by faboweb** [comment]
Tooo many changes in one PR ^^
The explore thing looks good. But the PendingUndelegations thing looks clumsy. Want to remove that and put it into the backend?

**Comment by Bitcoinera** [comment]
> Tooo many changes in one PR ^^
> The explore thing looks good. But the PendingUndelegations thing looks clumsy. Want to remove that and put it into the backend?

Done. Now I am directly querying for the pending undelegations. This query exists in the PR #232, `livepeer explore mode new api`.

I can also take this bit out and make another PR only with the pending undelegations thing, although it is not so much of a change for the FE.

File: src/components/staking/Undelegations.vue [hunk 3]
```diff
@@ -35,18 +33,11 @@ export default {
...
       query() {
         /* istanbul ignore next */
-        return gql`
-          query pendingundelegations($networkId: String!, $delegatorAddress: String!) {
-            pendingundelegations(networkId: $networkId, delegatorAddress: $delegatorAddress) {
-              ${UndelegationFragment}
-            }
-          }
-        `
+        return UndelegationsForDelegator(this.network)
```
Comment (faboweb) [inline src/components/staking/Undelegations.vue]: we only use this query here. what do you think about putting the query in here directly like we do at other places?

Comment (Bitcoinera) [reply src/components/staking/Undelegations.vue]: Done!

**Pull request merged**